graphql_compiler/compiler/ir_lowering_gremlin/ir_lowering.py

```diff
@@ -148,16 +153,19 @@ def validate(self):
                     u'Allowed types are {}.'
                     .format(type(block), self.folded_ir_blocks, allowed_block_types))
 
-        if not isinstance(self.field_type, GraphQLList):
-            raise ValueError(u'Invalid value of "field_type", expected a list type but got: '
+        if isinstance(self.field_type, GraphQLList):
```

Comment:
Can the field_type be null?

Reply:
Not in this location, which is why a `None` value will fall through to the `else` clause and raise an error. There are some situations where the type of the field doesn't really fit in with our type definitions (e.g. `@this` or `@rid` in OrientDB), but this is not one of them.

Reply:
Can it be a GraphQLNonNull type?

Reply:
We do not yet support GraphQLNonNull types in the compiler, so it shouldn't be. I added a call to `strip_non_null_from_type()` just in case, though.